integrations/integration-vip-config.php

```diff
@@ -115,39 +115,42 @@ public function is_active_via_vip(): bool {
 	}
 
 	/**
-	 * Get site status.
+	 * Get integration status for site.
 	 *
-	 * @return string|null
+	 * For single sites simply return global status.
+	 * For multisites, try to get status based on current blog id,
+	 * if not found then fallback to global environment status.
 	 *
-	 * @private
+	 * @return string|null
 	 */
 	public function get_site_status() {
 		if ( $this->get_value_from_config( 'org', 'status' ) === Org_Integration_Status::BLOCKED ) {
 			return Org_Integration_Status::BLOCKED;
```

Comment:
Should we get rid of the status level differentiation here as enums while we're at it?

Reply:
I think probably going to end up removing org information entirely.